src/api/exportable-lib/index.js

```diff
@@ -24,6 +36,14 @@ export default {
         return builder.getFunction();
     },
 
+    RequestLogger,
+
+    RequestMock,
+
+    RequestHook,
+
+    ResultPromise,
```

Comment:
I feel we need to discuss why this thing is exported.

Reply:
Motivation:
Requests are asynchronous. It's impossible to check result of custom request hook without adding timeouts. For example, try to remove `PromiseResult` for this test - https://github.com/DevExpress/testcafe/pull/2200/files#diff-4b5df15c8ad0e18c0978bf61152150ceR33

It means that customer cannot create a custom request hook with convenient API.